Dockerfile

```diff
@@ -36,21 +36,21 @@ ARG PROXY_UNOPTIMIZED
 ARG PROXY_FEATURES
 
 RUN --mount=type=cache,target=/var/lib/apt/lists \
-    --mount=type=cache,target=/var/tmp \
-    apt update && apt install -y time cmake
+  --mount=type=cache,target=/var/tmp \
```

Comment:
Auto formatting here? Not a blocker but just wondering if this was intentional.

Reply:
yup, i think this change can be removed entirely — this was left over from an attempt at feature-flagging.

Reply:
Yeah, would prefer to revert this as it removes some indentation below